src/main/java/uk/gov/hmcts/reform/professionalapi/configuration/SecurityConfiguration.java

```diff
@@ -58,7 +58,8 @@ public void setAnonymousPaths(List<String> anonymousPaths) {
     @Override
     public void configure(WebSecurity web) {
         web.ignoring()
-                    .antMatchers(anonymousPaths.toArray(new String[0]));
+                    .antMatchers(anonymousPaths.toArray(new String[0]))
+                    .antMatchers("/refdata/external/v1/organisations/mfa");
```

Comment:
I believe we do not want any web security like User and Service tokens around this API. Please let me know if there is some other scenario.

Reply:
that's right, this is only for MFA - GET api, the rest should always need authentication.

Reply:
Thanks Ishrat.